test/controllers/JourneyAnswersControllerSpec.scala

```diff
@@ -33,31 +33,36 @@ class JourneyAnswersControllerSpec extends ControllerBehaviours with ScalaCheckP
     expensesService = StubExpensesAnswersService()
   )
 
-  "JourneyAnswersController" should {
-    s"return a $NO_CONTENT when saveIncomeAnswers" in forAll(incomeJourneyAnswersGen) { data =>
+  "saveIncomeAnswers" should {
```

Comment:
Could we maybe add some stand-in UTs for the error case? I know we are just sending a 500 each time, but I guess when we change this to implement proper error handling the tests will fail so we don't forget to add extra cases